packages/shared/src/components/notifications/NotificationItemAvatar.tsx

```diff
@@ -40,7 +40,6 @@ function NotificationItemAvatar({
           user={{
             id: referenceId,
             username: referenceId,
-            name,
```

Comment:
This was expected? (it has a lot of implementation and might unwanted break something else

Reply:
It's not used in the component it's being passed to, so don't think it will break anything